xarray/tests/test_dask.py

```diff
@@ -473,7 +476,6 @@ def test_from_dask_variable(self):
         self.assertLazyAndIdentical(self.lazy_array, a)
 
 
-@requires_dask
 @pytest.mark.parametrize("method", ['load', 'compute'])
```

Comment:
@jhamman For future reference, it looks like `pytest.mark.parametrize` doesn't play well with the unittest based `skiptest` decorator. So I switched the import check to use something different for this module, but it's something we'll have to watch out for in the future, too.